donkeycar/parts/tfmini.py

```diff
@@ -0,0 +1,70 @@
+import time
+import serial
+
+class TFMini:
+    """
+    Class for TFMini and TFMini-Plus distance sensors.
+    See wiring and installation instructions at https://github.com/TFmini/TFmini-RaspberryPi
+
+    Returns distance in centimeters. 
+    """
+
+    def __init__(self, port="/dev/serial0", baudrate=115200, poll_delay=0.01, init_delay=0.1):
+        self.ser = serial.Serial(port, baudrate)
+        self.poll_delay = poll_delay
+
+        self.dist = 0
+        self.strength = 0
```

Comment:
This member variable is not really used. Does it make sense to return it - is this valuable information? Below you only take readings if `strength > 0`. Otherwise we should probably remove it.